code/modules/jobs/job_exp.dm

```diff
@@ -0,0 +1,186 @@
+// Admin Verbs
+
+/client/proc/cmd_admin_check_player_exp()	//Allows admins to determine who the newer players are.
+	set category = "Admin"
+	set name = "Check Player Playtime"
+	if(!check_rights(R_ADMIN))
+		return
+	var/msg = "<html><head><title>Playtime Report</title></head><body>Playtime:<BR><UL>"
+	for(var/client/C in GLOB.clients)
+		msg += "<LI> - [key_name_admin(C)]: <A href='?_src_=holder;getplaytimewindow=\ref[C.mob]'>" + C.get_exp_living() + "</a></LI>"
+	msg += "</UL></BODY></HTML>"
+	src << browse(msg, "window=Player_playtime_check")
+
+
+/datum/admins/proc/cmd_show_exp_panel(var/client/C)
+	if(!C)
+		to_chat(usr, "ERROR: Client not found.")
+		return
+	if(!check_rights(R_ADMIN))
+		return
+	var/body = "<html><head><title>Playtime for [C.key]</title></head><BODY><BR>Playtime:"
+	body += C.get_exp_report()
+	body += "</BODY></HTML>"
+	usr << browse(body, "window=playerplaytime[C.ckey];size=550x615")
+
+
+// Procs
+
+
+/datum/job/proc/available_in_playtime(client/C)
```

Comment:
bad name.
```required_playtime_remaining(client/C)```

```available_in_playtime(client/C)``` reads like it returns a bool, it doesn't, it returns 0-SOME AMOUNT, which actually if you were to use it like a bool (as you've done in preferences) would be the opposite of how it appears!

```if(available_in_playtime(client))``` will only succeed if it's NOT available!

tl;dr bad name.